modules/tinymce/src/core/main/ts/paste/CutCopy.ts

```diff
@@ -92,10 +93,10 @@ const cut = (editor: Editor) => (evt: EditorEvent<ClipboardEvent>): void => {
           // Restore the range before deleting, as Chrome on Android will
           // collapse the selection after a cut event has fired.
           editor.selection.setRng(rng);
-          editor.execCommand('Delete');
+          DeleteCommands.deleteCommand(editor, caret);
```

Comment:
Maybe we need to add a comment here why we call this like this directly. That it's to avoid selection normalization, undo levels, focus and other things that a command usually does.
So no one goes in and tries to fix this at some future point. 😃